build/make-olm.sh

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+CSV_VERSION="${1:-0.0.0}"
+BUNDLE_DIR="manifests/node-maintenance-operator/"
+BUNDLE_DIR_VERSION="${BUNDLE_DIR}${CSV_VERSION}/"
+
+
+# TODO: tested with operator-sdk 0.7.0: should we require it?
+which operator-sdk &> /dev/null || {
+	echo "operator-sdk not found (see https://github.com/operator-framework/operator-sdk)"
```

Comment:
It's nice that you are using operator-sdk :+1:

Reply:
all credit goes to @fromanirh :)